README.md

```diff
@@ -109,6 +109,73 @@ hdt.fromFile('./test/test.hdt')
   });
 ```
 
+### Reading the header
+HDT supports reading the header as string using `document.readHeader()`.
+The example below reads the header as string, and parses the header using the [N3.js](https://github.com/RubenVerborgh/N3.js/) library.
+
+```JavaScript
+var N3 = require('n3');
+var doc;
+var parser = N3.Parser();
+hdt.fromFile('./test/test.hdt')
+  .then(function(hdtDocument) {
+    doc = hdtDocument;
+    return doc.readHeader();
+  })
+  .then(function(header) {
+    var triples = [];
+    return new Promise(function(resolve, reject) {
+      parser.parse(header, function(error, triple) {
+        if (error) return reject(error);
+        if (triple) return triples.push(triple);
+        resolve(triples);
+      });
+    });
+  })
+  .then(function(triples) {
+    console.log('Read triples from header:\n', triples);
+  })
+  .catch(function(e) {
+    console.error(e);
+  })
+```
+### Writing a new header
+To write header information to an HDT, use `document.writeHeader(toFile, header)`, that returns an HDT document of the output file.
```

Comment:
This name seems to be (slightly) misleading, given that we're not writing a header but an entire HDT file. How about `changeHeader`? I'd also be inclined to swap the argument order.